manage.py

```diff
@@ -36,6 +37,26 @@ def add_event_identifier():
         save_to_db(event)
 
 
+@manager.command
+def switch(system):
+    import time
+    keys = [i.name for i in Module.__table__.columns][1:]
+    convey = {'True':'On', 'False':'Off'}
+    if system == 'all':
+        for key in keys:
+            value = getattr(Module.query.first(), key)
+            time.sleep(1)
+            setattr(Module.query.first(), key, not value)
+            db.session.commit()
+            print("Module %s turned %s" % (key, convey[str(not value)]))
+    else:
+        if system in keys:
+            value = getattr(Module.query.first(),system)
+            time.sleep(1)
```

Comment:
Is the system so fast that we deliberately need to make it slow? Or is it needed for some reason

Reply:
I apologize for pushing this as it was making sense with some previous functionality here, similar mistake may not be repeated.

Reply:
Removed